scribe.go

```diff
@@ -42,59 +42,117 @@ func (s *Scribe) new(t Type, msg string, data interface{}) {
 }
 
 // Notification will create a new notificaton entry
-func (s *Scribe) Notification(msg string, data interface{}) {
+func (s *Scribe) Notification(msg string) {
+	s.new(TypeNotification, msg, nil)
+}
+
+// Notificationf will create a new notificaton entry with a format message
+func (s *Scribe) Notificationf(msg string, args ...interface{}) {
+	msg = fmt.Sprintf(msg, args...)
+	s.Notification(msg)
+}
+
+// NotificationWitData will create a new notificaton entry with Data
+func (s *Scribe) NotificationWitData(msg string, data interface{}) {
```

Comment:
typo: `s/NotificationWitData/NotificationWithData`

Reply:
Great catch, thank you :)